Pack3/games/AwShirt/content/AwShirtSlogans.jet

```diff
@@ -992,7 +992,7 @@
         },
         {
             "id": 40567,
-            "suggestion": "Wiya fìtrr fe' leru",
+            "suggestion": "Pak, fìtrr fe' leru",
```

Comment:
We only have examples of pak at the end of a sentence. So I guess just "Fìtrr pak" could work

Reply:
Because of that i used the "," 
So it should be only "Fìtrr pak"?

Reply:
Sran